ui/components/app/incoming-trasaction-toggle/incoming-transaction-toggle.test.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import { within } from '@testing-library/dom';
```

Comment:
Why are we not using this react testing library ?

 import { within } from '@testing-library/react';
```suggestion
import { within } from '@testing-library/react';
```